ledger/metrics/collectd/collectd-original.conf

```diff
@@ -0,0 +1,2101 @@
+#
```

Comment:
Why check this file in?

Where did it come from?

Reply:
It shouldn't be checked in, I have removed it. It's the default config that was useful as a reference when writing my own config.